tools/firmware.toit

```diff
@@ -547,7 +547,7 @@ class ImageHeader:
     if marker != MARKER_: throw "image has wrong marker ($(%x marker) != $(%x MARKER_))"
     return image[0..HEADER_SIZE_]
 
-/*
+/**
```

Comment:
I think this comment is not supposed to be interpreted by toitdoc. -> Keep it as is.